[23기 최정민]Add : 로그인 페이지 레이아웃

minmin9324 opened the pull request:
>wecode PR message template 및 체크 리스트 입니다. 
아래 사항들을 전부 작성/체크 하시고 PR 해주세요!

## 수정 사항 간략한 한줄 요약
(여기에 수정 사항에 대한 간략한 한줄 요약/제목 작성해 주세요.)
- 로그인 페이지 레이아웃, westagram feed페이지 레이아웃

		 
## 수정 사항들 자세한 내용
(여기에 수정 사항에 대한 자세한 내용을 작성해 주세요.)
- id '@' 포함, pw 5 글자 이상 입력 시 버튼 활성화(예시)

## 기타 질문 및 특이 사항
(궁금한 사항들, 하면서 느낀 점들 공유해주실 것이 있으면 자유롭게 작성해 주세요.)
- 함수 로직이 너무 긴 것 같습니다. 좀 더 효율적인 방법이 없을지 궁금합니다.(예시) 

## 체크 리스트 (아래 사항들이 전부 체크되어야만 merge가 됩니다!)
- [ ] 필요한 test들을 완료하였고 기능이 제대로 실행되는지 확인 하였습니다.
- [ ] Wecode의 코드 스타일 가이드에 맞추어 코드를 작성 하였습니다.
- [ ] 제가 의도한 파일들과 수정 사항들만 커밋이 된 것을 확인 하였습니다.
- [ ] 본 수정 사항들을 팀원들과 사전에 상의하였고 팀원들 모두 해당 PR에 대하여 알고 있습니다.

joonsikyang commented on students/10th/ohogeun/js/main.js:
```diff
@@ -0,0 +1,77 @@
+    
+    let commentLast = document.getElementsByClassName('commentBox')[num];
+    let value = document.getElementsByClassName('commentInput')[0].value;
+
+    
+
```
불필요한 엔터는 삭제해주세요! 한 줄이면 충분할 것 같습니다 :)

dhgh95 replied:
넵

joonsikyang commented on students/10th/ohogeun/login.html:
```diff
@@ -0,0 +1,37 @@
+              <div class="loginLog">
+                  <img alt="인스타 로고" src="img/logo_text.png" />
+              </div>
+
```
16, 28번째 줄처럼 HTML에 불필요한 엔터는 삭제해주세요! 보기 좋을 수 있겠지만 class명으로, 혹은 태그로 이미 구분이 되어 있기 때문에 한 줄 띄우는 경우는 많이 없습니다. (요소 혹은 섹션에 대한 설명이 필요하면 주석으로 표시해주세요!)

dhgh95 replied:
넵

joonsikyang commented on students/10th/ohogeun/main.html:
```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>westargram</title>
+  <link href="style/common.css" rel="stylesheet" type="text/css" />
+  <link href="style/main.css" rel="stylesheet" type="text/css" />
+</head>
+
```
이 부분도 불필요한 엔터는 삭제해주세요!

dhgh95 replied:
넵

joonsikyang commented on students/10th/ohogeun/style/login.css:
```diff
@@ -0,0 +1,49 @@
+    font-size: 14px;
+    color: #00376B;
+    font-weight: bold;
+ }
```
전체적으로 클래스명 적용 잘해주셨네요! 이해하기 좋았습니다.

dhgh95 replied:
감사합니다!

joonsikyang commented on students/10th/ohogeun/style/login.css:
```diff
@@ -0,0 +1,49 @@
+
+.forgetPw{
+    margin-top: 125px;
+    font-size: 14px;
+    color: #00376B;
+    font-weight: bold;
```
font 속성끼리는 붙여서 작성해주세요! 항상 관련도, 우선 순위 따져서 나름의 convention을 갖고 작성하는 습관을 기르는 것이 좋습니다.

dhgh95 replied:
네

dhgh95 commented on students/10th/ohogeun/js/login.js:
```diff
@@ -0,0 +1,13 @@
+    const idBox = document.getElementsByClassName("loginId")[0].value;
+    const pwBox = document.getElementsByClassName("loginPw")[0].value;
+
+    if(idBox && pwBox){
+        button.style.backgroundColor = "blue";
+    }else if (!(idBox && pwBox)) {
+        button.style.backgroundColor = "#B2DFFC";
+    }
```
삼항연산자로 변경했습니다

dhgh95 commented on students/10th/ohogeun/js/main.js:
```diff
@@ -0,0 +1,77 @@
+
+search.addEventListener('focus', function(){
+    const searchText = document.getElementsByClassName('searchText')[0];
+    const searchImg = document.getElementsByClassName('searchImg')[0];
+    
+    searchText.style.display = "none";
+    searchImg.style.left = "5px";
+    search.placeholder = "검색"
```
placeholder를 새로운 클래스를 add, remove하여  focus off => 중앙정렬, focus => 왼쪽정렬로  변경하였습니다.

dhgh95 commented on students/10th/ohogeun/js/main.js:
```diff
@@ -0,0 +1,77 @@
+    const searchText = document.getElementsByClassName('searchText')[0];
+    const searchImg = document.getElementsByClassName('searchImg')[0];
+    
+    searchText.style.display = "none";
+    searchImg.style.left = "5px";
```
넵

jamessoun93 commented on students/10th/ohogeun/js/login.js:
```diff
@@ -4,10 +4,6 @@ loginBox.addEventListener('keyup', function () {
-        button.style.backgroundColor = "#B2DFFC";
-    }
+    let buttonColor = button.style.backgroundColor;
+    (idBox && pwBox) ? button.style.backgroundColor = "blue" : button.style.backgroundColor = "#B2DFFC"
```
`button.style.backgroundColor` 의 값만 변하는 것이기 때문에 값 부분에만 삼항연산자를 적용하시면 됩니다~
```suggestion
    button.style.backgroundColor =  (idBox && pwBox) ? "blue" : "#B2DFFC"
```

dhgh95 replied:
넵

jamessoun93 commented on students/10th/ohogeun/js/main.js:
```diff
@@ -4,37 +4,26 @@ let num = 0;
+    const searchImg = document.getElementsByClassName('searchImg')[0];   
+    searchImg.classList.add("searchImgFocus")
+    search.classList.add("searchFocus")
+    console.log("a")
```
개발단계에서 사용하신 console.log() 문은 push 할 때 꼭 빼주세요!

dhgh95 replied:
넵

jamessoun93 commented:
LGTM 이 얼마 남지 않았습니다 💪💪💪 
마지막 수정사항만 반영해서 push 해주세요!

joonsikyang commented:
LGTM👍

Commit:
로그인 레이아웃

Commit:
Add:로그인,메인 페이지 기능 추가 , 메인페이지  레이아웃 수정
 로그인페이지 로그인 기능 추가 - id와 pw에 1글자 이상 input되면 로그인 버튼 활성화
 메인페이지 게시버튼 활성화 기능 추가 - 1글자 이상 input시 게시 버튼 활성화
 메인페이지 댓글 게시 기능 추가  - 게시 버튼 클릭 or Enter키 누를시 댓글 달림
 메인페이지 레이아웃 추가 -  댓글달기창, 게시 버튼 추가
 메인페이지 아이콘이미지 오류  수정 - 네비게이션에 프로필 아이콘  src 오타 수정
 로그인 js파일 변수 선언 let에서  const로 수정

Commit:
modify : 로그인 페이지 , 메인페이지 변수명 수정,bottom-up 수정
- 메인페이지 -  필요없는 태그 삭제 진행중
-메인페이지 - 변수명 수정 중
-메인페이지 - 필요없는 css속성 삭제 중
-메인페이지 - css순서 수정 중
-로그인페이지 - 변수명 수정, 필요없는 css수정, 일부  태그 hight 속성 삭제, css 순서수정  완료
t message for your changes. Lines starting
